src/renderer/components/select/select.scss

```diff
@@ -46,16 +46,15 @@ html {
     }
 
     &__control {
-      border: none;
+      border: solid 1px $halfGray;
```

Comment:
And again, we are modifying global component styles without any explanation. Probably out-of-scope for this PR?

Reply:
I don't remember about this one. The other style changes I remember being needed for `<Wizard>` but I don't remember if this one way, will revert and check.